pandas/core/window/ewm.py

```diff
@@ -584,12 +586,14 @@ def _cov(X, Y):
                     X,
                     start,
                     end,
-                    self.min_periods,
+                    # error: Argument 4 to "ewmcov" has incompatible type
+                    # "Optional[int]"; expected "int"
+                    self.min_periods,  # type: ignore[arg-type]
```

Comment:
in the outer scope we have 

```
            min_periods = (
                self.min_periods
                if self.min_periods is not None
                else window_indexer.window_size
            )
```
can we not just pass min_periods here.

Reply:
cc @mroeschke ?

Reply:
True; since the EWM constructor always has an numeric `min_periods` this can just be `self.min_periods`